ceph.spec.in

```diff
@@ -152,7 +152,8 @@ BuildRequires:	fuse-devel
 # enough.
 BuildRequires:	devtoolset-8-gcc-c++ >= 8.3.1-3.1
 %else
-BuildRequires:	gcc-c++
+# at least gcc9 is needed. If not available, use gcc (this requires rpm>=4.13)
+BuildRequires:	(gcc-c++ >= 9 or gcc9-c++)
```

Comment:
did you mean `or gcc-c++` ?

Reply:
no. `gcc-c++` should be equal or greater than 9 if not, try `gcc9-c++`.